src/components/Form.js

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Is this Form also for stretch goals?  A lil confused why the only form on the page is named Select and there's an empty Form here